config/runtime.exs

```diff
@@ -116,7 +116,7 @@ purge_cron =
     ],
     else: []
 
-usage_tracking_cron = [{"0 */1 * * *", Lightning.UsageTracking.Worker}]
+usage_tracking_cron = [{"0 2 */1 * *", Lightning.UsageTracking.Worker}]
```

Comment:
Very unlikely we would have an hour of downtime but if it happens between 2 and 3, would you mind checking what would happen with current setup?

Reply:
@jyeshe If we miss the 02:00 slot, no metrics will be sent until the next day. This may or may not be a problem but it is probably too early in the process to tell. 

As we are making this opt-out, I thought that once per day is probably a 'polite' level for non-OpenFN intallations. If Taylor would like a more frequent poll for the OpenFN instance, then we should probably make this configurable.

Reply:
WYT as a failover to schedule it for 04:00 and make it idempotent?